pkgs/tools/misc/sheldon/default.nix

```diff
@@ -11,16 +11,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sheldon";
-  version = "0.7.3";
+  version = "0.7.4";
 
   src = fetchFromGitHub {
     owner = "rossmacarthur";
     repo = pname;
-    rev = version;
-    hash = "sha256-vGFR8NL3bOCUuNr0KQuAbjQMxvFbN/T9aVmf7Wxt9JU=";
+    rev = "9a56dcfc79c6e9f9bcf8480c162c50db013bb172";
```

Comment:
```suggestion
    rev = version;
```

Reply:
@sikmir @seqizz I thought that having "more stable" variable like a hash here is better. Do I miss smt here?

Reply:
What do you mean "more stable"? git tag is a such one.

Reply:
never mind, hash will cover this case